tests/test_truth_preflight.py

```
@@ -0,0 +1,635 @@
+                    "DEEPSCAN_API_TOKEN": "tok",
+                    "DEEPSCAN_OPEN_ISSUES_URL": url_value
+                },
+            )
+        url = args.args[0]
+        kwargs = args.kwargs
+        self.assertEqual(url, url_value)
+        self.assertEqual(kwargs["allowed_host_suffixes"], {"deepscan.io"})
+        self.assertEqual(kwargs["headers"]["Authorization"], "Bearer tok")
+
+    def test_env_resolved_url_provider_without_url_is_unreadable_not_silent(
+        self, ) -> None:
```

Flake8 (7.3.0)

[error] continuation line with same indent as next logical line

(E125)